packages/lib/src/core/Errors/ErrorPanel.tsx

```diff
@@ -0,0 +1,45 @@
+import { h } from 'preact';
+import './ErrorPanel.scss';
+
+export interface ErrorPanelObj {
+    errorMessages: string[];
+    fieldList: string[];
+}
+
+export interface ErrorPanelProps {
+    id?: string;
+    heading?: string;
+    errors: ErrorPanelObj;
+    callbackFn?: (who) => void;
+    showPanel?: boolean;
+}
+
+export function ErrorPanel({
+    id = 'ariaConsolidatedErrorField',
+    heading = 'Errors:',
+    errors,
+    callbackFn = null,
+    showPanel = false
+}: ErrorPanelProps) {
+    if (!errors) return null;
+
+    const { errorMessages } = errors;
+
+    // Perform passed callback, if specified
+    callbackFn?.(errors);
```

Comment:
Is it intentional to leave the callback here instead of putting it inside an `useEffect` function?

Every time that set state happens, this component likely will render and the callback will be called

Reply:
Good point - we have a check, in the callback, that prevents the callback from running again - so I wasn't noticing it being called excessively... but this is better